crit/images/handler.go

```diff
@@ -113,8 +113,6 @@ func ProtoHandler(magic string) (proto.Message, error) {
 		return &SignalfdEntry{}, nil
 	case "SK_QUEUES":
 		return &SkPacketEntry{}, nil
-	case "STATS":
-		return &StatsEntry{}, nil
```

Comment:
Hi @snprajwal, it would be good to keep the Go version of CRIT compatible with the Python version.
I think we can refactor crit to allow images to be imported as separate packages to avoid the problem with increased binary size.